index.js

```diff
@@ -4278,6 +4278,20 @@
   //. Takes a string and returns Just the date represented by the string
   //. if it does in fact represent a date; Nothing otherwise.
   //.
+  //. As noted in [#488][], this function's behaviour is unspecified for some
```

Comment:
It might be good to explicitly note that we are using the Date constructor to perform the parsing. 
```
Takes a string and returns Just the date represented by the string if it does in fact represent a date, as determined by the native `Date` constructor; Nothing otherwise.
```

Reply:
Thanks, Brad. Excellent suggestion. I'll make that change.